<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>471</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout" rowstretch="1,5">
     <item row="0" column="0">
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
       <item>
        <widget class="QLabel" name="dayInWeek_label">
         <property name="text">
          <string>Day in Week</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="date_label">
         <property name="text">
          <string>Date</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QListWidget" name="listWidget"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>0</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="notifier_layout"/>
   </widget>
   <widget class="QPushButton" name="setting_pushButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="achivements_pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>achivements</string>
    </property>
   </widget>
   <widget class="QPushButton" name="lineChart_pushButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>10</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>514</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
